.gitignore

```diff
@@ -4,4 +4,5 @@
 .astro
 dist/
 tmp/
-docs/public/llm.txt
\ No newline at end of file
+docs/public/llm.txt
+*~
```

Comment:
I don't mind this addition, but while you are here, could you leave a trailing newline as well.